cli/src/commands/explore/storage.rs

```diff
@@ -0,0 +1,176 @@
+use crate::utils::type_description::print_type_description;
+use crate::utils::type_example::print_type_examples;
+use crate::utils::{print_docs_with_indent, with_indent};
+use clap::Args;
+
+use std::fmt::Write;
+use std::write;
+
+use color_eyre::eyre::eyre;
+use frame_metadata::v15::{PalletMetadata, PalletStorageMetadata, StorageEntryType};
+
+use scale_info::form::PortableForm;
+
+use subxt::OnlineClient;
+use subxt::{config::SubstrateConfig, Metadata};
+
+#[derive(Debug, Clone, Args)]
+pub struct StorageSubcommand {
+    storage_entry: Option<String>,
+    #[clap(required = false)]
+    trailing_args: Vec<String>,
+}
+
+pub(crate) async fn explore_storage(
+    command: StorageSubcommand,
+    metadata: &Metadata,
+    pallet_metadata: &PalletMetadata<PortableForm>,
+    custom_online_client_url: Option<String>,
+) -> color_eyre::Result<()> {
+    let pallet_name = pallet_metadata.name.as_str();
+    let trailing_args = command.trailing_args.join(" ");
+    let trailing_args = trailing_args.trim();
+
+    let Some(storage_metadata) = &pallet_metadata.storage else {
+        println!("The \"{pallet_name}\" pallet has no storage entries.");
+        return Ok(());
+    };
+
+    // if no storage entry specified, show user the calls to choose from:
+    let Some(entry_name) = command.storage_entry else {
+        let storage_entries = print_available_storage_entries(storage_metadata, pallet_name);
+        println!("Usage:\n    subxt explore {pallet_name} storage <STORAGE_ENTRY>\n        view details for a specific storage entry\n\n{storage_entries}");
```

Comment:
nit:

```suggestion
        println!("Usage:");
        println!("    subxt explore {pallet_name} storage <STORAGE_ENTRY>");
        println!("        view details for a specific storage entry\n\n{storage_entries}");
```

just an example of another place multiple lines makes sense for me